packages/extension/public/_locales/en/messages.json

```diff
@@ -0,0 +1,205 @@
+{
+  "topSitesUsage": {
+    "message": "Top sites is based on your site usage"
+  },
+  "topSites": { "message": "Top Sites" },
+  "takeABreak": { "message": "Take a break" },
+  "activelyViewedFor": {
+    "message": "The site {{site}} was actively viewed for"
```

Comment:
I think across the board on this we need to update the variable interpolation.

https://developer.chrome.com/extensions/i18n-messages#example